### PR TITLE
docs: Add NodeJS SDK docs

### LINKED ISSUE #3640: NodeJS Documentation
### What is the issue?

We will need the following for the NodeJS launch.
- New section in docs
- Get Started Guide
- API reference guide
- Installation
- FAQ

@slumbering  can help support as needed since he is NodeJS SDK release lead.

### PR DESCRIPTION
**Pull request by vikram-dagger**
This commit adds documentation for the NodeJS SDK.

Closes #3640 
Signed-off-by: Vikram Vaswani <vikram@dagger.io>

### CODE REVIEW

**Comment by vikram-dagger** [comment]
Marked as draft since these are stubs atm.

File: docs/current/sdk/nodejs/835948-install.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+slug: /sdk/nodejs/835948/install
+---
+
+# Installation
```
Comment (vikram-dagger) [inline docs/current/sdk/nodejs/835948-install.md:5]: Are we recommending yarn or npm? Do we already have the package name?

Comment (slumbering) [reply docs/current/sdk/nodejs/835948-install.md:5]: I would suggest both npm and yarn

Comment (aluzzardi) [reply docs/current/sdk/nodejs/835948-install.md:6]: > Are we recommending yarn or npm?

Either both or if only one that should be `npm`

> Do we already have the package name?

`@dagger.io/dagger` to be consistent with `dagger.io/dagger` (Go) and `dagger-io` (Python)

**Comment by vikram-dagger** [comment]
Closing this PR as it was incorrectly rebased

**Pull request closed without merging**